src/main/java/com/echobox/api/linkedin/types/posts/Post.java

```diff
@@ -189,6 +195,51 @@ public class Post extends LinkedInURNIdType {
   @LinkedIn
   private Visibility visibility;
   
+  /**
+   * Mapping of image URN to image details.
+   * The image details will need to be retrieved via a separate API call and populated manually.
+   */
+  @Getter
+  @Setter
+  private Map<URN, ImageDetails> images;
+
+  public String getTitle() {
+    Content content = this.getContent();
+    if (content == null) {
+      return null;
+    }
+    
+    if (content.getMedia() != null) {
+      return content.getMedia().getTitle();
+    }
+    
+    if (content.getArticle() != null) {
+      return content.getArticle().getTitle();
+    }
+    
+    return null;
+  }
+  
+  public String getDescription() {
+    Content content = this.getContent();
+    if (content == null) {
+      return null;
+    }
+    
+    if (content.getArticle() == null) {
+      return null;
+    }
+    
+    return content.getArticle().getDescription();
+  }
+  
+  public List<String> getImageURLs() {
+    return images.values().stream()
+        .map(ImageDetails::getDownloadUrl)
+        .filter(StringUtils::isNoneBlank)
```

Comment:
```suggestion
        .filter(StringUtils::isNotBlank)
```
Why don't we just use isNotBlank? as downloadUrl is a simple String only

Reply:
done